README.md

```diff
@@ -166,6 +166,20 @@ It defaults to `setup-uv-cache` in the `TMP` dir, `D:\a\_temp\uv-tool-dir` on Wi
     cache-local-path: "/path/to/cache"
 ```
 
+### Disable cache pruning
+
+By default, the cache is pruned after a run, which means that all pre-built wheels are removed from
+the cache ([documentation](https://docs.astral.sh/uv/concepts/cache/#caching-in-continuous-integration)).
+If you want to keep the cache after a run, you can disable cache pruning with the `prune-cache` input.
```

Comment:
The documentation linked to says this:

>However, in continuous integration environments, persisting pre-built wheels may be undesirable. With uv, it turns out that it's often faster to omit pre-built wheels from the cache (and instead re-download them from the registry on each run). On the other hand, caching wheels that are built from source tends to be worthwhile, since the wheel building process can be expensive, especially for extension modules.

Reply:
Also, in my case I was wanting to run act locally but my connection quality is mediocre. This will greatly speed up local testing. Is there a way to dynamically set `prune-cache` depending on act vs Github Actions?

Reply:
```suggestion
By default, the uv cache is pruned after every run, removing pre-built wheels, but retaining any wheels that were built from source. On GitHub-hosted runners, it's typically faster to omit those pre-built wheels from the cache (and instead re-download them from the registry on each run). However, on self-hosted or local runners, preserving the cache may be more efficient. See the ([documentation](https://docs.astral.sh/uv/concepts/cache/#caching-in-continuous-integration)) for more.

If you want to persist the entire cache across runs, disable cache pruning with the `prune-cache` input.
```